domain/network/errors/errors.go

```diff
@@ -0,0 +1,11 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package errors
+
+import "github.com/juju/errors"
+
+const (
+	// ErrAlreadyExists is returned when a space already exists.
+	ErrAlreadyExists = errors.ConstError("already exists")
```

Comment:
can you add the domain as part of the string. i.e what isn't found. This is because strings are comparable so
```
ErrAlreadyExists = errors.ConstError("already exists")
ErrSimonIsGreat = errors.ConstError("already exists")
```

Both of these are comparable to the same thing.